lang/python/python-requests/Makefile

```diff
@@ -15,28 +15,52 @@ PKG_LICENSE:=Apache-2.0
 PKG_SOURCE:=requests-$(PKG_VERSION).tar.gz
 PKG_SOURCE_URL:=https://files.pythonhosted.org/packages/source/r/requests
 PKG_HASH:=502a824f31acdacb3a35b6690b5fbf0bc41d63a24a45c4004352b0242707598e
-PKG_BUILD_DIR:=$(BUILD_DIR)/requests-$(PKG_VERSION)
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(BUILD_VARIANT)-requests-$(PKG_VERSION)
 
 PKG_CPE_ID:=cpe:/a:python-requests:requests
 
 include $(INCLUDE_DIR)/package.mk
 include ../python-package.mk
+include ../python3-package.mk
 
-define Package/python-requests
+PKG_UNPACK:=$(HOST_TAR) -C $(PKG_BUILD_DIR) --strip-components=1 -xzf $(DL_DIR)/$(PKG_SOURCE)
+
+define Package/python-requests/Defaults
   SUBMENU:=Python
   SECTION:=lang
   CATEGORY:=Languages
   MAINTAINER:=Eneas U de Queiroz <cote2004-github@yahoo.com>
-  TITLE:=HTTP library for Python
   URL:=http://python-requests.org/
-  DEPENDS:=+python +chardet +python-idna +python-urllib3 +python-certifi
+endef
+
+define Package/python-requests
+$(call Package/python-requests/Default)
+  TITLE:=HTTP library for Python
+  DEPENDS:=+PACKAGE_python-requests:python +PACKAGE_python-requests:chardet +PACKAGE_python-requests:python-idna +PACKAGE_python-requests:python-urllib3 +PACKAGE_python-requests:python-certifi
```

Comment:
nitpick: in this case, it's neater to split on multiple lines like this:
```
 DEPENDS:= \
      +PACKAGE_python-requests:python \
      +PACKAGE_python-requests:chardet \
      +PACKAGE_python-requests:python-idna \
      +PACKAGE_python-requests:python-urllib3 \
      +PACKAGE_python-requests:python-certifi
```

Reply:
otherwise this looks good
@cotequeiroz 
comments ?

Reply:
Why not. It does look better.

Reply:
Fine with me.  The reason why I hadn't added a python3 build was because it was missing some dependencies, but I think you have it covered now.